galaxy_library_export.py

```diff
@@ -480,6 +496,7 @@ def ba(variableName, description, defaultValue=False):
 			[['--tags'], ba('tags', 'user tags')],
 			[['--themes'], ba('themes', 'game themes')],
 			[['--playtime'], ba('playtime', 'time spent playing the game')],
+			[['--LastPlayed'], ba('lastPlayed', 'last time the game was played')],

```

Comment:
Rename the parameter in `--last-played`, all lower case and with a dash, to keep consistency.